include/hip/hcc_detail/hip_runtime_api.h

```diff
@@ -1363,6 +1371,61 @@ hipError_t hipMemcpyDtoHAsync(void* dst, hipDeviceptr_t src, size_t sizeBytes, h
 hipError_t hipMemcpyDtoDAsync(hipDeviceptr_t dst, hipDeviceptr_t src, size_t sizeBytes,
                               hipStream_t stream);
 
+__attribute__((visibility("hidden")))
```

Comment:
I think the inline implementation of these functions are for HIP/HCC runtime only. Since this header file is shared between HIP/VDI runtime and HIP/HCC runtime, it is better to move them to a separate header file, e.g. hip_module_hcc.hpp. Then add conditions, e.g.

`
#if __hcc__
#include "hip_module_hcc.hpp"
#else
// keep original declarations 
#endif
`